### ISSUE TITLE
Change the source to a proper distribution

### ISSUE THREAD

**Issue by qw3ry** [techwizrd/fishmarks#19]
While the current install mechanism as mentioned in the README is good for a quick and dirty install, creating a distribution package will generally require the scripts to be installed in system directories.
Fish does provide mechanisms for this!

I'd recommend the following source structure:
- `<function_name>.fish` for each included function. This can be installed to `pkg-config --variable functionsdir fish` which points to `/usr/share/fish/vendor_functions.d`
- `fishmarks.fish` containing the aliases and the `SDIR` check. This can be installed to `pkg-config --variable confdir fish` (`/usr/share/fish/vendor_conf.d`)

This would clean up the installs and probably simplify the packaging process.
I am willing to provide a PR, if desired.

**Comment by techwizrd** [comment]
@qw3ry Are you still willing to provide a PR?

**Comment by qw3ry** [comment]
No, sorry. It has been a while and actually I'm not using fishmarks anymore. Feel free to close this
